ci/azure/posix.yml

```diff
@@ -69,20 +69,13 @@ jobs:
       displayName: 'Build versions'
 
     - task: PublishTestResults@2
+      condition: succeededOrFailed()
```

Comment:
Is there a particular reason to specify this instead of keeping as always?

Reply:
My understand was that `suceeded()` is the default condition on Azure as per [docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/conditions?view=azure-devops&tabs=yaml):

```
By default, steps, jobs, and stages run if all previous steps/jobs have succeeded. It's as if you specified "condition: succeeded()" (see Job status functions).
```

And I thought we want to publish test results regardless of whether failures are produced? Or am I missing something?

Reply:
My mistake. I guess ultimately my question is this versus `always` if it matters

Reply:
Thanks for taking a look! Apparently `always()` will run even if the pipeline is cancelled -> that's the difference